SPECS/kubevirt/Cleanup-housekeeping-cgroup-on-vm-del.patch

```diff
@@ -0,0 +1,266 @@
+From 5c2786d2968f150ce8725e2bac60b6b1e0e0a488 Mon Sep 17 00:00:00 2001
+From: Matt Jeanneret <mjeanneret@microsoft.com>
+Date: Thu, 9 Feb 2023 16:37:09 +0000
+Subject: [PATCH] Merged PR 28216: Cleanup housekeeping cgroup on vm delete
+
+Move cgroup.threads back to parent cgroup and rmdir the
+housekeeping cgroup before the vm is deleted.
+
+This prevents a bug in runc where it cannot read housekeeping/cgroup.procs
+stopping vm from being deleted
+
+Related work items: #502971
+---
+ Makefile                                     | 15 ++++++-
+ pkg/virt-handler/cgroup/cgroup.go            |  6 +++
+ pkg/virt-handler/cgroup/cgroup_v1_manager.go | 19 +++++++++
+ pkg/virt-handler/cgroup/cgroup_v2_manager.go | 19 +++++++++
+ pkg/virt-handler/cgroup/util.go              | 33 +++++++++++++++
+ pkg/virt-handler/vm.go                       | 44 ++++++++++++++++++++
+ 6 files changed, 135 insertions(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index 3d1a722f9..51ea5ae4c 100644
+--- a/Makefile
```

Comment:
this makefile change isnt needed.  its completely specific to the afo build environment

Reply:
nevermind, i was looking at an old copy